build.gradle

```diff
@@ -394,6 +394,9 @@ project(':iceberg-flink-runtime') {
   dependencies {
     compile project(':iceberg-flink')
     compile project(':iceberg-aws')
+    compile(project(':iceberg-nessie')) {
+      exclude group: 'com.google.code.findbugs', module: 'jsr305'
+    }
```

Comment:
Do we need to update the LICENSE and NOTICE for the Jar?

Reply:
doh, thanks for the reminder. Fixed